documentation/content/Components/photoswipe-instagramp-gallery.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: Gallery -- Use Instagram
+authors: Talha Mansoor, Pablo Iranzo Gómez
+Tags: nuances, images, gallery, instagram
+Date: 2020-02-06
+Slug: photoswipe-gallery-using-instagram
+Category: Components
+---
+
+Similar to the use case defined in [Photogallery]({filename}photogallery.md), Pelican has code in place for showing instragram galleries.
+
+## Article contents
+
+To enable the plugin, just define a similar div inside your article for each gallery:
+
+```yaml
+---
+author: Pablo Iranzo Gómez
+title: Instagram
+tags: python, redken, descuenbot, deal, chollo, bargain, rabbate, soldes
+date: 2020-01-27 00:35:00 +0100
+comments: true
+category: blog
+description:
+lang: en
+---
+Single image
+<div class="elegant-instagram" data-instagram-id="B7yh4IdItNd"></div>
+Multiple image
+<div class="elegant-instagram" data-instagram-id="BwWo35fAcR3"></div>
+```
+
```

Comment:
Tell user class `elegant-instagram` is also important.

Reply:
done